source/1_engine/rex_engine/include/rex_engine/defines.h

```diff
@@ -29,6 +29,16 @@
     #error DEBUG_BREAK unsupported machine instruction ...
   #endif
 
+  //-------------------------------------------------------------------------
+  // End of line
+  #ifndef REX_EOL
+    #ifdef REX_PLATFORM_WINDOWS
+      #define REX_EOL "\r\n"
```

Comment:
I know they're used to init a string_view, but I don't know why we don't branch on platform there instead.
we want to avoid using defines as global constants as much as possible

Reply:
You want to move the global constants as well and use those instead of the defines?

Reply:
yea, I would've gone into that direction, I don't see a gain in splitting it up into a define and a global constant really
other than moving the branching to a different place